### DISCUSSION TITLE
Hey devs, React Native bar code scanner needs to be built.

### DISCUSSION THREAD

**Discussion by supolaris** [community/community#130870] [Mobile]
### Select Topic Area

Question

### Body

I can create bar code scanner with React Native Cli. What approach should be best I should follow as already built bar code package (npm) did not work for me. Tell me if I can get code help from you devs.

**Comment by Iron-Ham** [comment]
Hi @supolaris, 

This public community discussion is meant to center around GitHub – with the `Mobile` tag relating specifically to the GitHub Mobile apps for iOS and Android. You may want to open an issue with feedback for the folks working on `react-native` at the [appropriate repo](https://github.com/facebook/react-native/issues).


Thanks,
Hesham
